build.proj

```diff
@@ -78,11 +78,25 @@
   <ItemGroup>
     <TestFile Include="$(MSBuildThisFileDirectory)/test/data/external/winequality-white.csv"
       Url="https://archive.ics.uci.edu/ml/machine-learning-databases/wine-quality/winequality-white.csv"
-      DestinationFile="$(MSBuildThisFileDirectory)test/data/external/winequality-white.csv" />
-    
-    <TestFile Condition="'$(IncludeBenchmarkData)' == 'true'" Include="$(MSBuildThisFileDirectory)/test/data/external/WikiDetoxAnnotated160kRows.tsv"
+      DestinationFile="$(MSBuildThisFileDirectory)test/data/external/winequality-white.csv" />    
+  </ItemGroup>
+  
+  <ItemGroup Condition="'$(IncludeBenchmarkData)' == 'true'" >    
+    <TestFile Include="$(MSBuildThisFileDirectory)/test/data/external/WikiDetoxAnnotated160kRows.tsv"
```

Comment:
The duplication here could be simplified using MSBuild. Something along the lines of:

```XML
<TlcResourceFile Include="WikiDetoxAnnotated160kRows.tsv" />
<TlcResourceFile Include="MSLRWeb10KTrain3.6MRows.tsv" />
<TlcResourceFile Include="MSLRWeb10KValidate1.2MRows.tsv" />
<TlcResourceFile Include="MSLRWeb10KTest1.2MRows.tsv" />

<TestFile Include="$(MSBuildThisFileDirectory)/test/data/external/%(TlcResourceFile.Identity)"
               Url="[http://aka.ms/tlc-resources/benchmarks/%(TlcResourceFile.Identity)](http://aka.ms/tlc-resources/benchmarks/%(TlcResourceFile.Identity))"
               DestinationFile="$(MSBuildThisFileDirectory)test/data/external/%(TlcResourceFile.Identity)" />
```

I'm not 100% sure it is better, but it reduces the number of times these URLs need to be copied.

Reply:
That's batching, I think it would only work within a `<Target>` ?

Reply:
Yep dan is right. its not working in this case. any other suggestion here ?

Reply:
It works, you just need to have the right syntax.  I've updated the above with actual MSBuild code that works.

Reply:
Okay thanks :)